vumi_http_retry/app.py

```diff
@@ -29,6 +54,22 @@ def respond(cls, req, data, code=http.OK):
         req.setResponseCode(code)
         return json.dumps(data)
 
-    @app.route('/health')
+    @app.route('/health', methods=['GET'])
     def route_health(self, req):
         return self.respond(req, {})
+
+    @app.route('/requests/', methods=['POST'])
+    @inlineCallbacks
+    def route_requests(self, req):
+        # TODO validation
+        # TODO rate limiting
+        data = json.loads(req.content.read())
+
+        yield add_request(self.redis, self.config.redis_prefix, {
+            'owner_id': req.getHeader('x-owner-id'),
```

Comment:
Could also add it now, if that would be better? If you don't have a preference, I'd like to do that with the other validation stuff.